docs/guide/networking/ipv4-exhaustion-content.md

```diff
@@ -15,7 +15,7 @@ You can use other ranges, but before you use those ranges in your virtual networ
 
 - Shared address space defined by RFC 6598 for carrier-grade network address translation (NAT) that's treated as private address space in Azure Virtual Network. The address block is 100.64.0.0/10.
 - Public, internet-routable IP addresses that your organization doesn't own. This practice is discouraged because resources in the virtual network can’t access internet endpoints that are exposed over the public IP addresses.
-- Special-purpose address blocks that are defined by IANA, like 192.0.0.0/24, 192.0.2.0/24, 192.88.99.0/24, 198.18.0.0/15, 198.51.100.0/24, 203.0.113.0/24, 233.252.0.0/24, and 240.0.0.0/4.
+- Special-purpose address blocks that are defined by IANA, like 192.0.0.0/24, 192.0.2.0/24, 192.88.99.0/24, 198.18.0.0/15, 198.51.100.0/24, 203.0.113.0/24 and 233.252.0.0/24.
```

Comment:
I don't think removing this is accurate, I can create a Virtual Network just fine with this range.

```http
PUT /subscriptions/<sub>/resourceGroups/rg-test/providers/Microsoft.Network/virtualNetworks/vnet-test?api-version=2023-04-01

{
    "location": "eastus",
    "properties": {
        "addressSpace": {
            "addressPrefixes": ["240.0.0.0/4"]
        },
        "subnets": []
    }
}
```

```
GET /subscriptions/<sub>/resourceGroups/rg-test/providers/Microsoft.Network/virtualNetworks/vnet-test?api-version=2023-04-01

{
  "name": "vnet-test",
  "id": "/subscriptions/<sub>/resourceGroups/rg-test/providers/Microsoft.Network/virtualNetworks/vnet-test",
  "etag": "W/\"662028aa-2462-40c8-a884-c89b67ce42d9\"",
  "type": "Microsoft.Network/virtualNetworks",
  "location": "eastus",
  "properties": {
    "provisioningState": "Succeeded",
    "resourceGuid": "39c010e4-456e-4915-aba2-7ab13e9f2604",
    "addressSpace": {
      "addressPrefixes": [
        "240.0.0.0/4"
      ]
    },
    "subnets": [],
    "virtualNetworkPeerings": [],
    "enableDdosProtection": false
  }
}
```

I know the portal blocks it, but it's not actually disallowed at the service level.

What am I missing here @ramandhillon84?

Reply:
We do not want users to use 240.0/4 range in a Vnet, as this range will NOT work in a VNet for windows clients.

Reply:
Then the text needs to change above, which isn't stating this.  If you can word more clearly about what is POSSIBLE vs what is RECOMMENDED.  This section is a list of possibilities.  And if you're running Linux, it won't matter.  Please come up with a different way to say this than just removing that range.

Reply:
Additional changes, please see lines 19-21

Reply:
Thanks, that's great!